docs/docs/30-how-to-guides/15-working-with-freight.md

```diff
@@ -138,7 +120,21 @@ kargo update freight \
 ```
 
 This can also be accomplished via `kubectl` commands `apply`, `edit`, `patch`,
-etc.
+etc. by updating the `alias` field of the `Freight` resource.
+
+:::info
+The `alias` field is a convenient way to update the `Freight` resource's
+`kargo.akuity.io/alias` label, which causes a webhook to sync the field value
+to the label value. The precedence rules for syncing between the field and
+label values are as follows:
+
+- If the field is specified, the webhook syncs the value to the label value.
+- If the field is unspecified, the webhook syncs the label value to the field
+  value.
```

Comment:
This feels backwards to me.

> If the field is specified, the webhook syncs the value to the label value.

This sounds to me like the label's value is assumed by the field.

> If the field is unspecified, the webhook syncs the label value to the field
  value.

This sounds to me like the label assumes the field's value.

I think "sync" might be interpreted differently by native English speakers v ESL speakers, or may even interpreted differently _within_ each of those groups.

I propose the following:

```suggestion
- If the field has a non-empty value, the label will assume the field's value.
- If the field has an empty value, the field will assume the label's value.
  value.
```

Reply:
Reading it twice, it also actually feels backwards to me now as well due to the _"**to** the [..] value"_.

I will update it with your suggestions to avoid this.